### PR TITLE
Chores for v257

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: test/units/TEST-13-NSPAWN.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ fi
 # shellcheck source=test/units/test-control.sh
 . "$(dirname "$0")"/test-control.sh
 
+export TEST_SKIP_SUBTESTS="nspawn machined"
```
Comment (yuwata) [inline test/units/TEST-13-NSPAWN.sh:16]: I cannot find any failures recently for nspawn. Do you?

Comment (bluca) [reply test/units/TEST-13-NSPAWN.sh:16]: There was one last week, unfortunately I forgot to save the logs locally and retention is just a few days due to size

File: po/be.po [hunk 2]
```diff
@@ -1262,8 +1262,8 @@ msgstr "Для перазапуску \"$(unit)\" патрабуецца аўт
 
 #: src/core/dbus-unit.c:566
 msgid ""
-"Authentication is required to send a UNIX signal to the processes of "
-"'$(unit)'."
+"Authentication is required to send a UNIX signal to the processes of '$"
+"(unit)'."
```
Comment (keszybz) [inline po/be.po]: Please drop the commit that "updates" the po files. There are no changes except some that lines are wrapped differently. This doesn't seem to be caused by a change in the strings, but instead a different version of the gettext tools. The existing formatting was arguably better.

**Review by keszybz — CHANGES_REQUESTED** [review]
Yeah, let's do this. But please drop that one commit.

**Comment by keszybz** [comment]
For the sake of expediency, I force-pushed to drop one commit.

File: po/be.po (hunk 2 above)
Comment (bluca) [reply po/be.po]: is there some config that can be set? it's annoying having to manually understand what to commit and what to skip

Comment (yuwata) [reply po/be.po]: I recently run the command, but I did not observe that kind of changes. Also, the line wrap looks quite spurious. I agree with dropping the commit.

**Comment by keszybz** [comment]
ci ubuntu/noble: linux-headers-generic : Depends: linux-headers-6.8.0-50-generic but it is not installable
ci opensuse/tumbleweed: /var/tmp/rpm-tmp.i6mRxE: line 6: /usr/sbin/sysusers2shadow: No such file or directory

Same old.

**Comment by bluca** [comment]
Actually the first one is new, but unrelated

File: test/units/TEST-13-NSPAWN.sh (hunk 1 above)
Comment (yuwata) [reply test/units/TEST-13-NSPAWN.sh:16]: Hm, OK.

**Comment by bluca** [comment]
This is a new test:

```
[  104.813998] TEST-07-PID1.sh[4126]: + sleep .5
[  105.317801] TEST-07-PID1.sh[4165]: ++ systemctl show -P SubState TEST-07-PID1-private-pid.service
[  105.332095] TEST-07-PID1.sh[4126]: + [[ exited != \f\a\i\l\e\d ]]
[  105.332358] TEST-07-PID1.sh[4126]: + sleep .5
[  105.496938] TEST-07-PID1.sh[4073]: + at_exit
```

So I'll disable this one too https://github.com/systemd/systemd/issues/35528

**Comment by bluca** [comment]
Wait for what? It's flaky, it will take weeks to know if that helps or not

**Comment by keszybz** [comment]
Yeah, I think we should merge the test disablement. We can reverse it in main and stable later.

testing-farm:fedora-rawhide-x86_64: `84/92 systemd:integration-tests / TEST-75-RESOLVED                                              FAIL             26.98s   exit status 1`
Doesn't seem related. There were some DNS updates in RH infra, might be related to that.

**Pull request merged**